node/src/main/scala/com/wavesplatform/api/http/eth/EthRpcRoute.scala

```diff
@@ -56,10 +56,10 @@ class EthRpcRoute(blockchain: Blockchain, transactionsApi: CommonTransactionsApi
     } ~ (get & path("abi" / AddrSegment)) { addr =>
       complete(blockchain.accountScript(addr).map(as => ABIConverter(as.script).jsonABI))
     } ~ (pathEndOrSingleSlash & post & entity(as[JsObject])) { jso =>
-      val id             = (jso \ "id").get
-      val params         = (jso \ "params").asOpt[IndexedSeq[JsValue]].getOrElse(Nil)
-      lazy val param1    = params.head
-      lazy val param1Str = param1.as[String]
+      val id              = (jso \ "id").get
```

Comment:
An absence of "id" leads to `{"error":0,"message":"Error is unknown"}`. Is it intended behavior? I see:
```
params.headOption.toRight(complete(GenericError("Expected parameter not found")))
```
below, which looks better IMO.